src/main/resources/application-test.yml

```diff
@@ -0,0 +1,26 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5433/scheduler_application
```

Comment:
This should allow overrides I.E
jdbc:postgresql://${DB_HOST:localhost}:${DB_PORT:5433}/${DB_NAME:scheduler_application}

Reply:
done